website/docs/r/cosmosdb_cassandra_cluster.html.markdown

```diff
@@ -77,11 +77,49 @@ The following arguments are supported:
 
 * `default_admin_password` - (Required) The initial admin password for this Cassandra Cluster.
 
+* `authentication_method` - (Optional) The authentication method that is used to authenticate clients. Possible values are `None` and `Cassandra`. Defaults to `Cassandra`.
+
+* `client_certificate` - (Optional) An `client_certificate` block as defined below.
+
+* `external_gossip_certificate` - (Optional) An `external_gossip_certificate` block as defined below.
+
+* `external_seed_node` - (Optional) An `external_seed_node` block as defined below.
+
+* `identity` - (Optional) An `identity` block as defined below.
+
+* `version` - (Optional) The version of Cassandra what the Cluster converges to run. Possible values are `3.11` and `4.0`. Defaults to `3.11`. Changing this forces a new Cassandra Cluster to be created.
+
+* `repair_enabled` - (Optional) Is the automatic repair enabled on the Cassandra Cluster?
+
 * `tags` - (Optional) A mapping of tags assigned to the resource.
 
+---
+
+A `client_certificate` block supports the following:
+
+* `pem` - (Required) A list of TLS certificates that is used to authorize client connecting to the Cassandra Cluster.
+
+---
+
+A `external_gossip_certificate` block supports the following:
+
+* `pem` - (Required) A list of TLS certificates that is used to authorize gossip from unmanaged Cassandra Data Center.
+
+---
+
+A `external_seed_node` block supports the following:
+
+* `ip_address` - (Required) A list of IP Addresses of the seed nodes in unmanaged the Cassandra Data Center which will be added to the seed node lists of all managed nodes.
```

Comment:
would this make more sense as (also address should be addresses?)
```suggestion
A `external_seed_node_ip_addresses` - (Required) A list of IP Addresses of the seed nodes in unmanaged the Cassandra Data Center which will be added to the seed node lists of all managed nodes.
```

Reply:
@katbyte , if service team expands the sub properties for `external_seed_node`, seems currently it's not easily to support these new sub properties to TF. So suggest align with the property schema of service api. Does it make sense?

Reply:
do they have active plans to? it's easy enough to change it if we need to as there no bools here

Reply:
I am just curious on why servic team treats it as block. I suppose they intend to expand sub properties in the future.

Reply:
Updated Code.